README.md

```diff
@@ -139,17 +139,18 @@ a Python version greater than 2.7 (Python 3.2 is not supported anymore).
 * **Python** 2.7 or 3.3+
 * **PyQt5** 5.2+ or **PyQt4** 4.6+: PyQt5 is recommended.
 * **qtconsole**: Enhanced Python interpreter.
-* **Rope** and **Jedi**: Editor code completion, calltips
+* **Rope** and **Jedi**: Editor code completion, calltips.
```

Comment:
There's no need to put a dot at the end here. Please notice that the text continues in the next line :-)